<launch>
  <!-- (OPTIONAL) Put in the path to the dataset as a default value for the argument -->
  <arg name="path_to_dataset" />
  <arg name="descriptor" default="SIFT" />

  <node name="lab_5" pkg="lab_5" type="track_features" output="screen">
    <param name="img1_path" type="string" value="$(find lab_5)/images/box.png"/>
    <param name="img2_path" type="string" value="$(find lab_5)/images/box_in_scene.png"/>
    <param name="mode" type="int" value="1"/>
    <param name="descriptor" type="string" value="$(arg descriptor)"/>

    <remap from="/images_topic" to="/camera/color/image_raw"/>
  </node>
  <node name="rosbag" pkg="rosbag" type="play" output="screen"
  args="--clock $(arg path_to_dataset)  /tesse/left_cam/rgb/image_raw:=/camera/color/image_raw" />
</launch>
